server_installation/topics/cache/eviction.adoc

```diff
@@ -5,61 +5,31 @@ There are multiple different caches configured for {project_name}.
 There is a realm cache that holds information about secured applications, general security data, and configuration options.
 There is also a user cache that contains user metadata.  Both caches default to a maximum of 10000 entries and use a least recently used eviction strategy.
 Each of them is also tied to an object revisions cache that controls eviction in a clustered setup.
-This cache is created implicitly and has twice the configured size.
-There are also separate caches for user sessions, offline tokens, and login failures.  These caches are unbounded in size as well.
+This cache is created implicitly and has twice the configured size. There are some more caches for save the authorization data, keys etc.
+There are also separate caches for user sessions, offline tokens, login failures and other things. These caches are unbounded in size by default.
```

Comment:
The rewrite below is better. Even better yet would be if we could fill in details instead of saying "and so on" or "and other things."

This cache is created implicitly and has twice the configured size. There are also caches for the authorization data, keys, and so on. In addition, there are also separate caches for user sessions, offline tokens, login failures, and other things. These caches are unbounded in size by default.

Reply:
I've changed like this in my local fork and added some details. Although adding more details is always double-side coin as it means more documentation changes will be needed in the future when implementation changes. See my comment below. So I've used this:

This cache is created implicitly and has twice the configured size. The same applies for the `authorization` cache, which holds for
the authorization data. The `keys` cache holds data about external keys and doesn't need to have dedicated revisions cache. Rather
it has `expiration` explicitly declared on it, so the keys are forced to be periodically downloaded from external clients or
identity providers.

In addition, there are also separate caches for user sessions, client sessions, offline tokens, login failures, action tokens and other things.
These caches are unbounded in size by default.

Reply:
Perhaps we can create a list of all caches and put that list in an obvious place in the documentation, then refer to it as the canonical reference when we make comments like "and other things".

My fear is that we don't want to frustrate readers/users by being vague. I definitely also agree with you about the double-sided coin of more detail means more documentation changes. The balance is that we should provide the information our users need, and only that information when we are writing procedures, but we should also find a way to provide a complete reference somewhere to the other details that they may eventually need.

Reply:
Yes, I see. I agree with you that it is sometimes not easy to find the right balance.. In the end, I've added even more details into the "evictions" sections related to caches (talked today with @pedroigor about some things, which we're missing the docs). Also all the caches are used here and referenced from "replication" chapter.

Reply:
I like that approach, @mposolda. What do you think about replacing "and other things" with a reference like "and others, for more information about all of the caches, see ..."?